pkg/registry/consul.go

```diff
@@ -43,8 +43,9 @@ func NewConsulClient(config *ConsulConfig) (IRegistry, error) {
 // Register is used for node registration which creates a session to consul
 func (c *ConsulClient) Register(name string, ttl time.Duration) (string, error) {
 	sessionID, _, err := c.client.Session().Create(&api.SessionEntry{
-		Name: name,
-		TTL:  ttl.String(),
+		Name:      name,
+		TTL:       ttl.String(),
+		LockDelay: 2,
```

Comment:
from the documentation https://www.consul.io/docs/dynamic-app-config/sessions, it says
```
While the default is to use a 15 second delay, clients are able to disable this mechanism by providing a zero delay value.
```
Is 2s a good value for us?

Reply:
currently we have 5s retry period for reattempting the lock acquire so anything below 5 should be fine